packages/contentful/src/crn/types/webhook.ts

```diff
@@ -9,7 +9,8 @@ export type ContentfulWebhookPayloadType =
   | 'teams'
   | 'users'
   | 'workingGroups'
-  | 'tutorials';
+  | 'tutorials'
+  | 'news';
```

Comment:
we already have `news` here :)
<img width="522" alt="Screenshot 2023-11-29 at 09 35 05" src="https://github.com/yldio/asap-hub/assets/16595804/3312a336-ea0d-4909-88bb-cc92d77b7327">